server/routes/serviceProviderReferrals/actionPlanFormPresenter.test.ts

```diff
@@ -1,8 +1,8 @@
-import AddActionPlanActivitiesPresenter from './addActionPlanActivitiesPresenter'
```

Comment:
I'm a bit confused about the `addActionPlanActivities` -> `actionPlanForm` rename. Could you explain a little why you think it'll be useful to use the same presenter / view for multiple pages in the journey? We didn't do this for the referral journey because the pages didn't really have much in common. Curious to know why we think this journey is different.

In particular, does it make sense to have an `ActionPlanFormView` that renders a more specifically-named `serviceProviderReferrals/actionPlan/addActivities`?

Let's talk it through if it's easier that way.

Reply:
This was something @tomsmyers suggested last week, and introduced as part of his PR. I think I quite liked the approach as it's a small form with only 3 or so pages so felt quite nice to keep it self-contained.

I can see your point about the `view` though, I don't think it's quite right - a view should represent a single screen/page in the service. I guess it feels a bit out of sync to have a single `FormPresenter` but individual `view` objects so am happy to undo that commit (hence why I didn't squash it in yet!)

@tomsmyers do you have a strong preference on this, or any other reasons I've missed?

Reply:
Yeah, personally I don't understand the benefit - I think that the "one presenter per page" model is easy to understand, and that it makes sense to deviate from it if for example you've got a whole bunch of very similar pages, but I don't think that's the case here.

Reply:
no strong preference - just want to avoid repetition where possible.

i think the three pages in question are very similar though. they share everything but the input elements.

Reply:
Ok, as discussed have removed the last commit on this.